tests/unit/api/test_bootstrap.py

```diff
@@ -243,23 +299,6 @@ def test_post_bootstrap_invalid_token(self, test_client, monkeypatch):
 
         token_headers = {"Authorization": "Bearer h4ck3r"}
 
-        mocked_check_metadata = pretend.call_recorder(lambda: False)
```

Comment:
We can remove those mocks as we catch the empty payload in `bootstrap_state`, but before we didn't or what?